app/assets/stylesheets/base/_base.scss

```diff
@@ -13,3 +13,15 @@
 @import "lists";
 @import "tables";
 @import "typography";
+
+a {
+  .b {
+    .c {
+      .d {
+        .e {
+          color: blue;
```

Comment:
Color `blue` should be written in hexadecimal form as `#0000ff`<br>Color literals like `blue` should only be used in variable declarations; they should be referred to via variable everywhere else.